### PR TITLE
Added compatible wallets

### PR DESCRIPTION
**Pull request by Mirobit**

### CODE REVIEW

File: bip-0044.mediawiki [hunk 1]
```diff
@@ -262,12 +262,20 @@ is required and a pull request to the above file should be created.
...
 * [[https://play.google.com/store/apps/details?id=com.mycelium.wallet|Mycelium Bitcoin Wallet (Android)]] ([[https://github.com/mycelium-com/wallet|source]])
 * [[https://copay.io/|Copay]] ([[https://github.com/bitpay/copay|source]])
+* [[https://play.google.com/store/apps/details?id=com.bonsai.wallet32|Wallet32 @ Android]] ([[https://github.com/ksedgwic/Wallet32|source]])
 * [[https://maza.club/encompass|Encompass]] ([[https://github.com/mazaclub/encompass|source]])
 * [[https://www.coinvault.io/|CoinVault]] ([[https://github.com/CoinVault/dotblock|source]])
+* [[https://cancoin.co/|Cancoin]] ([[https://github.com/cancoin|source]])
+* [[https://samouraiwallet.com/|Samouria Wallet]] ([[https://github.com/Samourai-Wallet/samourai-wallet-android|source]])
+* [[https://www.darkwallet.is/|Darkwallet]] ([[https://github.com/darkwallet/darkwallet/|source]])
+
+* [[https://bitcointrezor.com/|TREZOR]] ([[https://github.com/keepkey/|source]])
```
Comment (saleemrashid) [inline bip-0044.mediawiki]: Source code links are mixed up (KeepKey source code for TREZOR and vice versa)

File: bip-0044.mediawiki [hunk 2]
```diff
@@ -262,12 +262,20 @@ is required and a pull request to the above file should be created.
...
+* [[https://www.keepkey.com/|KeepKey]] ([[https://github.com/trezor/|source]])
+* [[https://www.ledgerwallet.com/|Ledger Wallet]] ([[https://github.com/trezor/|source]])
```
Comment (saleemrashid) [inline bip-0044.mediawiki]: Source code link is incorrect, should be https://github.com/LedgerHQ

**Comment by luke-jr** [comment]
@slush0 @prusnak

**Comment by prusnak** [comment]
It's hard to find a new line that does not have a typo or factual error in it. NACK until all issues are fixed.

**Comment by Mirobit** [comment]
Fixed all mix-ups.

**Comment by saleemrashid** [comment]
@Mirobit I'm not sure about the opinion of others but I'd suggest squashing the commits

**Comment by Mirobit** [comment]
@saleemrashid I thought I already squashed the two commits. I only see one now.

File: bip-0044.mediawiki [hunk 3]
```diff
@@ -262,12 +262,20 @@ is required and a pull request to the above file should be created.
...
 * [[https://play.google.com/store/apps/details?id=com.mycelium.wallet|Mycelium Bitcoin Wallet (Android)]] ([[https://github.com/mycelium-com/wallet|source]])
 * [[https://copay.io/|Copay]] ([[https://github.com/bitpay/copay|source]])
+* [[https://play.google.com/store/apps/details?id=com.bonsai.wallet32|Wallet32 (Android)]] ([[https://github.com/ksedgwic/Wallet32|source]])
 * [[https://maza.club/encompass|Encompass]] ([[https://github.com/mazaclub/encompass|source]])
 * [[https://www.coinvault.io/|CoinVault]] ([[https://github.com/CoinVault/dotblock|source]])
+* [[https://cancoin.co/|Cancoin]] ([[https://github.com/cancoin|source]])
+* [[https://samouraiwallet.com/|Samourai Wallet]] ([[https://github.com/Samourai-Wallet/samourai-wallet-android|source]])
+* [[https://www.darkwallet.is/|Darkwallet]] ([[https://github.com/darkwallet/darkwallet/|source]])
+
+* [[https://bitcointrezor.com/|TREZOR]] ([[https://github.com/trezor/|source]])
```
Comment (saleemrashid) [inline bip-0044.mediawiki]: @jonathancross Yep, TREZOR.io is the new domain

File: bip-0044.mediawiki [hunk 4]
```diff
@@ -262,12 +262,20 @@ is required and a pull request to the above file should be created.
 
 ==Compatible wallets==
 
-* [[https://mytrezor.com|myTREZOR web wallet]] ([[https://github.com/trezor/webwallet|source]])
-* [[https://play.google.com/store/apps/details?id=com.bonsai.wallet32|Wallet32 @ Android]] ([[https://github.com/ksedgwic/Wallet32|source]])
 * [[https://play.google.com/store/apps/details?id=com.mycelium.wallet|Mycelium Bitcoin Wallet (Android)]] ([[https://github.com/mycelium-com/wallet|source]])
 * [[https://copay.io/|Copay]] ([[https://github.com/bitpay/copay|source]])
+* [[https://play.google.com/store/apps/details?id=com.bonsai.wallet32|Wallet32 (Android)]] ([[https://github.com/ksedgwic/Wallet32|source]])
```
Comment (prusnak) [inline bip-0044.mediawiki]: I would not include Wallet32 since it has not been updated since 2014.

File: bip-0044.mediawiki [hunk 5]
```diff
@@ -262,12 +262,20 @@ is required and a pull request to the above file should be created.
...
 * [[https://maza.club/encompass|Encompass]] ([[https://github.com/mazaclub/encompass|source]])
 * [[https://www.coinvault.io/|CoinVault]] ([[https://github.com/CoinVault/dotblock|source]])
+* [[https://cancoin.co/|Cancoin]] ([[https://github.com/cancoin|source]])
+* [[https://samouraiwallet.com/|Samourai Wallet]] ([[https://github.com/Samourai-Wallet/samourai-wallet-android|source]])
+* [[https://www.darkwallet.is/|Darkwallet]] ([[https://github.com/darkwallet/darkwallet/|source]])
+
+* [[https://bitcointrezor.com/|TREZOR]] ([[https://github.com/trezor/|source]])
+* [[https://www.keepkey.com/|KeepKey]] ([[https://github.com/keepkey/|source]])
+* [[https://www.ledgerwallet.com/|Ledger Wallet]] ([[https://github.com/LedgerHQ|source]])
+* [[https://21.co/learn/21-lib-wallet/|21 Machine Wallet]] ([[https://github.com/21dotco|source]])
```
Comment (prusnak) [inline bip-0044.mediawiki]: This is not a wallet for users, but rather developers and I don't think we should include these in the list.

Comment (saleemrashid) [inline bip-0044.mediawiki]: @prusnak Might be even more of a reason to include it (BIPs are aimed at developers)

**Comment by Mirobit** [comment]
@jonathancross Thanks. Removed and updated.

Also removed Darkwallet and Encompass due to inactivity. 
Added MultiBit (HD).

File: bip-0044.mediawiki (hunk 5 above)
Comment (Mirobit) [inline bip-0044.mediawiki]: Agree with @saleemrashid

**Comment by prusnak** [comment]
> Added MultiBit (HD).

Multibit is not a BIP-44 compatible wallet as it just supports one account and ignores the rest.

**Comment by luke-jr** [comment]
@Mirobit poke

**Comment by Mirobit** [comment]
@luke-jr removed MultiBit. Should be ready to merge now.

**Comment by prusnak** [comment]
ACK

**Review by Masoudahgl — COMMENTED** [review]
Like

**Pull request merged**